test/Confluent.Kafka.Avro.UnitTests/Configuration.cs

```diff
@@ -48,10 +48,10 @@ public void SerializerConfigure()
         {
             var avroSerializer = new AvroSerializer<int>(schemaRegistryClient);
 
-            var config = new Dictionary<string, object>
+            var config = new Dictionary<string, string>
             {
-                { "avro.serializer.buffer.bytes", 42 },
-                { "avro.serializer.auto.register.schemas", false }
+                { "avro.serializer.buffer.bytes", "42" },
```

Comment:
the auto conversion of types was pretty nice....

Reply:
but non-standard and the new way of doing things is to flip where the conversion happens to strongly typed config classes on top.